docs/all-of-equinox.md

```diff
@@ -48,7 +49,7 @@ y = jax.random.normal(y_key, (100, 2))
 model = MyModule(model_key)
 grads = loss(model, x, y)
 learning_rate = 0.1
-model = jax.tree_map(lambda m, g: m - learning_rate * g, model, grads)
+model = jtu.tree_map(lambda m, g: m - learning_rate * g, model, grads)
 ```
```

Comment:
Sorry, exception to the above: in the intro docs I just use `jax.whatever`, to avoid a long import list of `jax.random`, `jax.numpy` etc.

Reply:
sure, makes sense :).